src/semantic-ui/site/collections/table.overrides

```diff
@@ -65,27 +65,41 @@ Table Overrides - REACT-INVENIO-APP-ILS
   }
 }
 
-
 @media only screen and (max-width: @largestMobileScreen) {
+  .ui.popup.transition {
+    table {
```

Comment:
thanks for the fix! it will be easier to fix this by applying (:not) selector to the table below, as in 
```
table:not(.ui.popup.transition) { ...
```
it will reduce the amount of css